packages/components/src/components/button/button.scss

```diff
@@ -73,6 +73,16 @@
         .dc-btn__icon {
             color: var(--text-colored-background);
         }
+
+        &-dashboard {
+            background: var(--brand-apple-blue);
+            & .dc-btn__text {
+                color: var(--text-colored-background);
+            }
+            & .dc-btn--primary:hover {
+                background: var(--brand-apple-blue);
+            }
+        }
```

Comment:
Can we add a blue variant of the button rather than creating package-specific rules here?